use-cases/xtclim/pipeline.yaml

```diff
@@ -19,13 +21,39 @@ pipeline:
          preprocessing-split-step:
            class_path: preprocessing.preprocess_2d_seasons.SplitPreprocessedData
            init_args:
-               scenario: ${scenario} 
+               scenario: ${scenario}
          training-step:
-             #class_path: src.trainer_dist.XTClimTrainer
-           class_path: src.trainer.TorchTrainer
+           class_path: src.trainer.XTClimTrainer
            init_args:
                epochs: ${epochs}
                batch_size: ${batch_size}
                lr: ${lr}
-               #strategy: ${strategy}
-        
+               strategy: ${strategy}
+               # season is dynamically imported
+               logger:
+                 class_path: itwinai.loggers.LoggersCollection
+                 init_args:
+                   loggers:
+                     - class_path: itwinai.loggers.MLFlowLogger
+                       init_args:
+                         experiment_name: XTClim (Cerfacs)
+                         log_freq: epoch
+                     - class_path: itwinai.loggers.WandBLogger
+                       init_args:
+                         log_freq: epoch
+         evaluation-step:
```

Comment:
I am wondering whether the evaluation step would deserve a separate "inference pipeline" as we did for other use cases, or is just enough to execute it after the trainer